checks.d/docker.py

```diff
@@ -422,6 +422,9 @@ def _find_cgroup_filename_pattern(self):
                     return os.path.join('%(mountpoint)s/docker/%(id)s/%(file)s')
                 elif os.path.exists(stat_file_path_coreos):
                     return os.path.join('%(mountpoint)s/system.slice/docker-%(id)s.scope/%(file)s')
+                # Kubernetes, we can't test the existence of the path without the id
+                else:
```

Comment:
Not sure we want an else here since it acts as a `default`. An `elif` could be interesting too don't you think? This way if the path is not found because of a problem, we'll get an exception here instead of later in the execution.